### PR TITLE
apps/examples/smartfs: Revert changes related to append file operation

### PR DESCRIPTION
**Pull request by amogh-samsung**
- Append File Operation improvements are not guaranteed/indicated with the optimized buffer size.
- We keep read file operation improvement which is significant as a suggestion to the programmer.
- Modularity is retained considering the possible addition of any future improvement suggestions.

Signed-off-by: Amogh Hassija <a.hassija@samsung.com>

### CODE REVIEW

**Comment by amogh-samsung** [comment]
```
TASH>>smartfs_buf_opt
TASH>>
Correct test command usage is: smartfs_buf_opt <Timer Device No.> <File Size in Kb>
-> Read test example will read the file back to back, one buffer size at a time

TASH>>
TASH>>
TASH>>
TASH>>
TASH>>smartfs_buf_opt 0 6
TASH>>Timer path = /dev/timer0
Creating Test File
Test File Creation Successful
================================== STARTING READ TESTS ==================================


**************************************************************************************
Throughput while reading 6144Kb file 100 time(s) with 512byte buffer size = 990472.50Kbps

**************************************************************************************

**************************************************************************************
Throughput while reading 6144Kb file 100 time(s) with 1024byte buffer size = 1230103.88Kbps

**************************************************************************************
TEST WITH OPTIMIZED BUFFER

**************************************************************************************
Throughput while reading 6144Kb file 100 time(s) with 1013byte buffer size = 1518724.50Kbps

**************************************************************************************
================================== READ TESTS COMPLETE ==================================

Test Example Completed Successfully
SmartFs Optimized Buffer Test Example Exits


TASH>>
```

File: apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c [hunk 1]
```diff
@@ -42,8 +42,6 @@
 #define SAMPLE_FILE "/mnt/sample_file"
```
Comment (abhishek-samsung) [inline apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c:42]: Please change "operatino" to "operation" in the commit message.

Comment (amogh-samsung) [reply apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c:42]: Corrected, thanks.

File: apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c [hunk 2]
```diff
@@ -466,20 +312,11 @@ int buffer_optimization_main(int argc, char *argv[])
...
-	if (ret != OK) {
-		printf("Append test failed on file: %s, ret: %d\n", SAMPLE_FILE, ret);
-		ret = -1;
-	}
-
-error_with_open_timer_dev:
 	if (ioctl(frt_fd, TCIOC_STOP, 0) < 0) {
 		fprintf(stderr, "ERROR: Failed to stop the Free Run Timer: %d\n", errno);
 		ret = -1;
-		goto error_with_frt_fd;
```
Comment (thapav) [inline apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c:482]: Is this not required anymore ?

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by jeongarmy — APPROVED** [review]
Looks good to me

**Review by jsdosa — APPROVED** [review]
Looks nice to me.

File: apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c (hunk 2 above)
Comment (amogh-samsung) [reply apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c:482]: Was initially used to skip the append test if the read test fails
We have now omitted the append test itself.

**Review by thapav — APPROVED** [review]
LGTM

**Comment by dineshdas-samsung** [comment]
LGTM

**Review by Taejun-Kwon — APPROVED** [review]

**Pull request merged**